SparseGrids/TasmanianSparseGrid.cpp

```diff
@@ -454,6 +454,14 @@ void TasmanianSparseGrid::integrate(double q[]) const{
     }
 }
 
+void TasmanianSparseGrid::differentiate(const double x[], double jacobian[]) const {
+    if (isGlobal()) {
+        dynamic_cast<GridGlobal*>(base.get())->differentiate(x, jacobian);
```

Comment:
Check line 1980 in TasmanianSparseGrid.hpp, you can use:
```
get<GridGlobal>()->differentiate(x, jacobian);
```

Reply:
Changed.